app/components/Poll/Poll.css

```
@@ -63,61 +130,40 @@
   }
 }
 
-.pollHeader a {
-  font-weight: 500;
-  color: var(--lego-font-color);
-  line-height: 1.3;
-}
-
-.voteButton {
-  width: 100%;
-  margin: 1px !important;
+.resultsHiddenInfo {
+  font-style: italic;
 }
 
-.moreOptionsLink {
-  justify-content: space-between;
-}
-
-.blurContainer {
-  display: none;
-  position: absolute;
-  justify-content: center;
-  width: 100%;
-  height: 100%;
-  cursor: pointer;
-}
-
-.blurOverlay {
-  position: absolute;
-  z-index: 2;
+.totalVotes {
   font-weight: 500;
-  color: var(--color-absolute-white);
 }
 
-.optionWrapper:hover .blurContainer {
-  display: flex;
+.success {
+  color: var(--success-color);
 }
 
-.optionWrapper:hover .blurEffect {
-  filter: blur(3px);
-  pointer-events: none;
+.registrationCount {
+  margin-bottom: 0.5rem;
 }
 
-.totalVotes {
-  font-weight: 500;
+.bottomBar {
+  position: relative;
+  background-color: var(--lego-red-color);
+  width: 100%;
+  padding: 6.25rem 0 0.5rem;
+  top: -6rem;
+  margin-bottom: -6rem;
+  transition: 0.5s height, 0.5s padding;
+  z-index: 1;
 }
 
-@media (--mobile-device) {
-  .blurContainer {
-    display: flex;
-  }
-
-  .blurEffect {
-    filter: blur(3px);
-    pointer-events: none;
-  }
+.poll.expanded .bottomBar {
+  top: 0;
+  padding-top: 0.5rem;
+  margin-bottom: 0;
 }
 
-.success {
-  color: var(--success-color);
+.arrowIcon {
+  color: var(--lego-card-color);
+  text-shadow: 0 2px 2px rgba(0, 0, 0, 20%);
```

```suggestion
  text-shadow: var(--shadow-md);
```
Or a different one (most likely)